services/frontend-v3/src/components/profileForms/welcome/WelcomeView.jsx

```diff
@@ -167,6 +167,7 @@ const WelcomeView = ({
         </div>
       );
     }
+    // TODO GGGGGGGG THIS LOGIC NEEDS TO BE RETHINKED
```

Comment:
Love this comment haha (are you intending on changing it this PR or in the future or just forgot about it?)

Reply:
lol i was suppose to remove it and fix it.

There is a bunch of repeated logic in that file. will fix

Reply:
done